### PR TITLE
Breaking up creating subgraph flow

### PR DESCRIPTION
**Pull request by idalithb**
- Creating a Subgraph page is now a folder containing: `install-the-cli`;  `subgraph-manifest`; `ql-schema`; `assemblyscript-mappings`; `advanced`
- `install-the-cli` & `ql-schema` have content with some feedback implemented
- ALL pages need detailed edits moving forward

### CODE REVIEW

File: website/pages/en/developing/creating-a-subgraph/install-the-cli.mdx
Comment (lutter) [inline website/pages/en/developing/creating-a-subgraph/install-the-cli.mdx:17]: This is something that devs with more Javascript experience should weigh in on: we should tell people how to install these things, incuding actual commands to run, for popular platforms, or link to the instructions upstream. As somebody who has little JS experience, I always found these steps very annoying and intimidating.

I've also been using `pnpm` instead of `yarn` lately, and it seems nicer/faster. We should have a set of tools that we recommend and where we have good instructions for how to install them, and just mention the other ones.

But all this needs input from people who actually know what they are doing with JavaScript (not me)

File: website/pages/en/developing/creating-a-subgraph/install-the-cli.mdx
Comment (lutter) [inline website/pages/en/developing/creating-a-subgraph/install-the-cli.mdx:37]: Instead of a bulleted list, I would just make this a paragraph. I would just say that users can create a new subgraph from an existing contract or an example subgraph, and the the following subsections explain the details. I am not sure what the `--product subgraph-studio` flag is trying to tell me here, and it raises the question of what happens when I don't include that.

Not sure who's best equipped to answer that, either studio folks or Graph CLI devs?

**Review by lutter — COMMENTED** [review]
Splitting up the 'create subgraph' docs this way makes a lot of sense to me. I left a lot of comments on two of the files for places that need more work; I think the other files also need an in-depth review, but those two have already taken quite a bit of time, and we should schedule that one file at a time, but that shouldn't hold up these changes.

File: website/pages/en/developing/creating-a-subgraph/ql-schema.mdx
Comment (idalithb) [reply website/pages/en/developing/creating-a-subgraph/ql-schema.mdx:18]: @lutter Will do when I start detailed edits 😃

**Review by MichaelMacaulay — APPROVED** [review]
LGTM - amazing work!

**Pull request merged**
